### ISSUE TITLE
Little error on index

### ISSUE THREAD

**Issue by Zielin0** [Zielin0/linker#3]
```
TypeError: Failed to resolve module specifier ".prisma/client/index-browser". Relative references must start with either "/", "./", or "../".
```

**Comment by Zielin0** [comment]
It happens when you click on the Link in the `alias` column.

**Comment by Zielin0** [comment]
[index.vue:15](https://github.com/Zielin0/linker/blob/master/pages/index.vue#L15) might not need to be a type of `Link`.

**Comment by Zielin0** [comment]
Fixed @ [4aa41a47ee9e3526597f239b7922522c08340ff2](https://github.com/Zielin0/linker/commit/4aa41a47ee9e3526597f239b7922522c08340ff2)
